app/services/weather_service.rb

```diff
@@ -0,0 +1,24 @@
+class WeatherService
+
+  def self.connection
+    Faraday.new(
+      url: 'https://api.weather.gov/'
+    )
+  end
+
+  def self.parse(api_data)
+    JSON.parse(api_data.body, symbolize_names: true)
+  end
+
+  def self.all_weather_data(latitude, longitude)
+    response = connection.get("points/#{latitude},#{longitude}")
+    parse(response)
+  end
+
+  def self.forecast_data(latitude, longitude)
```

Comment:
Super awesome way to get this to work! Good job!

Reply:
Agreed this is great!